dotnet/src/InternalUtilities/src/Schema/KernelJsonSchemaBuilder.cs

```diff
@@ -33,9 +33,12 @@ internal static class KernelJsonSchemaBuilder
     [RequiresDynamicCode("Uses reflection to generate JSON schema, making it incompatible with AOT scenarios.")]
     public static KernelJsonSchema Build(Type type, string? description = null, JsonSchemaMapperConfiguration? configuration = null)
     {
+#pragma warning disable SKEXP0120
```

Comment:
Exposing non-experimental to experimental feature, suggest having a safe path for non-experimental existing API's.

Reply:
I'm not sure I follow, could you please elaborate what you mean?

Reply:
Elaborated offline

Reply:
Discussed internally and agreed that, in this case and in general, it's OK when non-experimental functionality depends on experimental functionality if the experimental functionality is not subject to change in a way that will alter behavior that will bubble up to the experimental API and breaks its behavior.